src/Icons/Models/IconLink.cs

```diff
@@ -99,16 +99,16 @@ public bool IsUsable()
             return false;
         }
 
-        if (Rel != null && _iconRels.Contains(Rel.Value.ToLower()))
```

Comment:
I did mean you can add that into the constructor of the `HashSet` but this is functionally equivalent so I'm fine with it if you are.

Reply:
ah my mistake, I forgot this _was_ a hashset